src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/styles.ts

```diff
@@ -50,3 +50,7 @@ export const ToggleCurrencyButton = styled.span`
   color: #464366;
   cursor: pointer;
 `
+
+export const MarketPriceDiff = styled.span<{ isPositive: boolean }>`
+  color: ${({ isPositive }) => (isPositive ? 'green' : 'red')};
```

Comment:
this misses the case when there is no price difference, maybe we want to show gray on that case.

Note: we have that on SimpleChart, would be great to start to reuse logic

Reply:
If the price has no difference we don't show it. So no need for a condition that isn't met